entrypoint.sh

```diff
@@ -270,6 +316,15 @@ test_go_ceph() {
     fi
     export CEPH_CONF
 
+    if [[ ${MIRROR_CONF} ]]; then
+      if [[ ${CEPH_VERSION} != nautilus ]]; then
```

Comment:
Could you move this check into the standard `setup_mirroring` function, so that we have simple blocks that we can easily cut out, when we remove this code? I would like all this code to be removable without even using my brain. ;-)

Reply:
Done!